src/TLuaInterpreter.cpp

```diff
@@ -3182,6 +3182,14 @@ int TLuaInterpreter::getOS(lua_State* L)
 #endif
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#getPID
+int TLuaInterpreter::getPID(lua_State* L)
+{
+    int pid = QApplication::applicationPid();
+    lua_pushinteger(L, pid);
+    return 1;
```

Comment:
Modern compilers will moan about `L` not being used - silence that with this change:
```suggestion
    Q_UNUSED(L)
    int pid = QApplication::applicationPid();
    lua_pushinteger(L, pid);
    return 1;
```